xapian-letor/docs/letor.rst

```diff
@@ -68,7 +68,6 @@ As mentioned before, this process requires a training file in the above format.
 Provided such information, API is capable of creating the training file which is in the mentioned format and can be easily used for learning a model. In xapian-letor we support the following learning algorithms:
 
     1. `ListNET <https://dl.acm.org/citation.cfm?id=1273513>`_
-    2. `Ranking-SVM <https://dl.acm.org/citation.cfm?id=775067>`_
```

Comment:
There's also documentation about libsvm in `xapian-letor/INSTALL` which should go as part of this commit.

It's a good idea when stripping out something like this to do something like `git grep -i svm` from the top of the source tree to see if there are any other lingering references to it - I just tried that and as well as `xapian-letor/INSTALL` I see some false positives in the stemming data, a mention in xapian-letor/ChangeLog (which is a historical reference so should stay), but also this which ought to go:

```
xapian-letor/tests/generate-api_generated:    'SVMRanker' => ''
```

Reply:
fixed in a fixup commit.